pkgs/by-name/wl/wl-clicker/package.nix

```diff
@@ -0,0 +1,50 @@
+{
+  lib,
+  fetchFromGitHub,
+  stdenv,
+  wayland-scanner,
+  wlr-protocols,
+  wayland,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "wl-clicker";
+  version = "0.3.1";
+
+  nativeBuildInputs = [ wayland-scanner ];
+  buildInputs = [
+    wlr-protocols
+    wayland
+  ];
+
+  src = fetchFromGitHub {
+    owner = "phonetic112";
+    repo = "wl-clicker";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-0ddi383ilxqm0nvn05asn6m77sgi96fw07np95wx089a97b0mr8y";
```

Comment:
Sorry, I didn't quite explain how to get the proper SRI hash.
You cannot use the base32 hash as-is, you can convert between them like so:

```console
$ nix-hash --to-sri --type sha256 0ddi383ilxqm0nvn05asn6m77sgi96fw07np95wx089a97b0mr8y
sha256-HuUK1kkqIdB5SdcewJ1J8elzqrFaFWC3BRV3GgcasTU=
```

Reply:
that's okay, i should've read up on it further. adjusted.